x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/table/degraded_docs_percentage_link.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiSkeletonRectangle, EuiFlexGroup, EuiLink } from '@elastic/eui';
+import React from 'react';
+import { useLinkToLogsExplorer } from '../../../hooks';
+import { QualityPercentageIndicator } from '../../quality_indicator';
+import { DataStreamStat } from '../../../../common/data_streams_stats/data_stream_stat';
+
+export const DegradedDocsPercentageLink = ({
+  isLoading,
+  dataStreamStat,
+}: {
+  isLoading: boolean;
+  dataStreamStat: DataStreamStat;
+}) => {
+  const logsExplorerLinkProps = useLinkToLogsExplorer({
+    dataStreamStat,
+    query: { language: 'kuery', query: '_ignored:*' },
+  });
+
+  return (
+    <>
```

Comment:
**nit:** This fragment shouldn't be necessary as `EuiSkeletonRectangle` is the only returned node